terraform/account/region/load_balancer_logs_create_table.tpl

```diff
@@ -0,0 +1,57 @@
+CREATE EXTERNAL TABLE IF NOT EXISTS {PICK YOUR ENVIRONMENT NAME} (
```

Comment:
Do we need to choose the env name or is this fine?

Reply:
we will when we use the query, but this is an account level resource and just a starter for ten. I may end up creating athena queries in the env level terraform per environment when I've written them as part of the investigation for Abbi.